### PR TITLE
Collectd for cs

### PR DESCRIPTION
**Pull request by benoit-canet**
Add cassandra stress monitoring

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -1186,7 +1189,11 @@ def node_setup(node):
             node.remoter.run('sudo yum install -y screen')
             if db_node_address is not None:
                 node.remoter.run("echo 'export DB_ADDRESS=%s' >> $HOME/.bashrc" %
-                                 db_node_address)
+                                     db_node_address)
```
Comment (lmr) [inline sdcm/cluster.py]: This small change makes the CI job to be unhappy. remember to use inspektor (`pip install inspektor`, `inspektor style --fix`) when writing the patches.

File: sdcm/cluster.py [hunk 2]
```diff
@@ -1209,6 +1216,8 @@ def node_setup(node):
         self.log.debug('Setup duration -> %s s', int(time_elapsed))
 
     def run_stress_thread(self, stress_cmd, timeout, output_dir, stress_num=1):
+
+        stress_cmd = stress_cmd + " -log file=/tmp/cs.log"
```
Comment (lmr) [inline sdcm/cluster.py]: I believe we don't need to output c-s log to an intermediate file. We can pipe the output of the process directly to the exporter (I'd reckon it's a good idea that the exporter pipes output from c-s to the stdout as well).

**Comment by benoit-canet** [comment]
@lmr Thanks a lot for the review I am iterating on it.

**Comment by kongove** [comment]
@benoit-canet any update for this?

**Comment by lmr** [comment]
@kongove I guess @benoit-canet is waiting for me to merge the code, which I'll do right away.

**Pull request merged**